recipes/r-dotcall64/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set version = '0.9-04' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-dotcall64
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: dotCall64_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/dotCall64_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/dotCall64/dotCall64_{{ version }}.tar.gz
+  sha256: 0e65a56c836aecd02380d9033989a05e391552eb6b3a9d16504a827b9faae662
+
+build:
+  number: 0
+  script: R CMD INSTALL --build .
+  skip: True  # [win32]
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - r-base
+    - posix                # [win]
+    - {{native}}toolchain  # [win]
+    - gcc                  # [not win]
+
+  run:
+    - r-base
```

Comment:
@jsignell we do need to list `libgcc  # [not win]` as a dependency to all packages that depend on `gcc`.